src/modules/uORB/topics/filtered_bottom_flow.h

```diff
@@ -33,34 +31,48 @@
  *
  ****************************************************************************/
 
-/**
- * @file filtered_bottom_flow.h
- * Definition of the filtered bottom flow uORB topic.
- */
+/* Auto-generated by genmsg_cpp from file /home/jgoppert/git/px4/Firmware/msg/filtered_bottom_flow.msg */
```

Comment:
Yes, absolutely, I missed that one in my earlier review. No generated files in GIT, of course!